.github/workflows/build-unified.yml

```diff
@@ -6,6 +6,13 @@ on:
     paths:
       - "docker-unified/**"
       - ".github/workflows/build-self-host.yml"
+  workflow_call:
```

Comment:
Is `workflow_call` needed here? As the `server` build is using `workflow_dispatch` event to trigger it, I think the `workflow_call` is not used: https://github.com/bitwarden/server/blob/master/.github/workflows/build.yml#L549

Reply:
Yes, we wanted to change the way the `build` workflow in `server` triggers this workflow.

Reply:
Okay